starter-code/src/functions-and-arrays.js

```diff
@@ -48,6 +124,34 @@ var wordsUnique = [
   'bring'
 ];
 
+
+function uniquifyArray (array) {
+  var newArray = [];
+    if(array.length === 0) {
+        return undefined;
+    }
+
+    else {
+      newArray.push (array[0]);
+
+      for (var i=1; i< array.length; i++){
```

Comment:
Using a for loop IN a for loop, is quite risky.. specially when you have a much more simple way to do it. 

You can use 2 methods:

indexOf : 
array.indexOf("Word") => will return the index of Word in the array if Word exists, or return -1 if the word doesn't exist. If it returns an index, you shouldnt push, if it returns -1 then you should push it.

Includes:
array.includes("Word") => will return TRUE if word exists, or FALSE if it doesnt exist. Push when its false. 

example with the indexOf :
```
for (var i = 0; i < array.length; i++) {
    if (newArray.indexOf(array[i]) === -1) {
      newArray.push(array[i]);
    }
  }
```

Hope it's clear :)